### PR TITLE
Folder setup

### PR DESCRIPTION
**Pull request by existentialcoder**
Check for the folder setup, linter, tester. Please give suggestions on how better things can be done. Lets finalise this structure before moving onto the functionalities @sampathBlam

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -0,0 +1,90 @@
+{
+  "name": "jsutil",
+  "version": "1.0.0",
+  "description": "JS Array and object utils",
```
Comment (sampathBlam) [inline package.json:4]:
```suggestion
  "description": "JS array and object utils",
```

**Review by sampathBlam — COMMENTED** [review]
@existentialcoder : can we rename the lib folder to utils and move it inside src folder ?

**Comment by sampathBlam** [comment]
Also, should we use npm or stick to yarn ? yarn is pretty outdated now. either we have to go to npm or experiment with yarn2. ur take ?

**Comment by existentialcoder** [comment]
> Also, should we use npm or stick to yarn ? yarn is pretty outdated now. either we have to go to npm or experiment with yarn2. ur take ? 

I have been using yarn for quite some time and its pretty stable only. Not sure which is outdated. Your call whether to go with npm or yarn 🧐

**Comment by existentialcoder** [comment]
> @existentialcoder : can we rename the lib folder to utils and move it inside src folder ?
> 

Done

File: package.json [hunk 2]
```diff
@@ -21,6 +21,10 @@
     }
   },
   "eslintConfig": {
+    "env": {
+      "node": true,
```
Comment (sampathBlam) [inline package.json:25]: @existentialcoder : should we set `browser: true` ? Our package can be useful for any frontend or backend js app, isn't it?

**Review by sampathBlam — APPROVED** [review]

**Pull request merged**